utils/scripts/organization.ts

```diff
@@ -69,7 +75,7 @@ export const getUserOrgData = async (orgID: number) => {
         owner: "",
         admins: [],
         }
-    if(orgData !== null){
+    if(orgData !== null && data !== undefined){
```

Comment:
Triggered an error for reading from null after leaving an organization and trying to access organization overview.

Steps to reproduce error:
```
- Leave organization
- Go to any other tab in the side nav bar
- Go back to Overview tab under Organization
```
Partly related to comment for `app/dashboard/organization/overview/page.tsx`. Page still shows "Create an Organization"

![organization overview error](https://github.com/nam-t24/Senior-CS-Project/assets/73014878/6af2d523-7306-4aea-a1c3-655237f4d4ab)